### DISCUSSION TITLE
Tab switching using arrow keys

### DISCUSSION THREAD

**Discussion by christian-photo** [zen-browser/desktop#2578] [Ideas]
Similar behavior was already implemented for workspaces, but I think it would be useful for tabs as well. Using some keyboard shortcut with the arrow keys you could go to the next or previous tab in the tab list.

**Comment by hastyscorpion** [comment]
Agree, Since this is like the "Firefox version of Arc Browser" one of the features that I really liked about it was that command option up arrow and command option down arrow would move you up and down your tabs list.

**Comment by nisutec** [comment]
A Hotkey to go to the next and previous tab would be great. The current hotkey options to select specific tabs are good if you are working with static tabs, but when you have many unstructured tabs, it would be great to have the option to cycle through all opened tabs with a key press.
